.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2.1
+
+jobs:
+  jdk8:
+    docker:
+      - image: circleci/openjdk:8-stretch
+    steps:
+      - checkout
+      - run: mvn verify -V
+
+  jdk11:
+    docker:
+      - image: circleci/openjdk:11-stretch
+    steps:
+      - checkout
+      - run: mvn verify -V
+
+  jdk15:
+    docker:
+      - image: circleci/openjdk:15-buster
```

Comment:
they do not provide the `stretch` version for JDK 15, so using `buster`

`stretch` is th eone used in all Java examples.